### PR TITLE
Fix/quickstart

### PR DESCRIPTION
**Pull request by SimonWallner**
added 1 sentence to link directly to quickstart sample

also added vscode devcontainer with recomennded plugins and spell checker

### CODE REVIEW

**Comment by SimonWallner** [comment]
another change to cloudpods coming up shortly...

**Comment by SimonWallner** [comment]
@HarshCasper ready now

File: content/en/getting-started/quickstart.md [hunk 1]
```diff
@@ -12,14 +12,16 @@ This quickstart guide will walk you through starting LocalStack on your local ma
 
 {{< alert title="Note" >}}
 The quickest way to experiment with LocalStack is to use one of our [LocalStack quickstart samples](https://app.localstack.cloud/quickstart) to deploy a thumbnail creator, request worker application, or an asynchronous microservice with tracing/debugging. The quickstart samples will automatically connect to your running LocalStack container on your local machine and run the applications.
+
+To quickly see the outcome of this quickstart guide, try running the [Request Worker](https://app.localstack.cloud/quickstart-cloudformation) sample from our website.
```
Comment (lukqw) [inline content/en/getting-started/quickstart.md:16]: `running` -> `deploying`

File: content/en/user-guide/tools/cloud-pods/_index.md [hunk 2]
```diff
@@ -9,14 +9,18 @@ aliases:
...
+Please note that cloud pods may not be compatible across different versions of LocalStack.
 We are actively working on addressing this limitation in the upcoming releases.
 {{< /alert >}}
 
 Cloud Pods are a mechanism that allows you to take a snapshot of the state in your current LocalStack instance, persist it to a storage backend, and easily share it with your team members.
 
 ![Persistence versus Cloud Pods](pods-persistence.png)
 
+{{< alert title="Demo">}}
+To quickly see cloud pods in action, we prepared a small sample that uses cloud pods to inject its state into a running LocalStack container. [Cloud Pod Thumbnail Sample App](https://app.localstack.cloud/quickstart/demo4)
```
Comment (lukqw) [inline content/en/user-guide/tools/cloud-pods/_index.md:21]: `a small sample that` -> `a small sample application`

**Comment by SimonWallner** [comment]
brought up to date with main

**Pull request merged**